android/modules/geolocation/src/java/ti/modules/titanium/geolocation/TiLocation.java

```diff
@@ -335,13 +335,12 @@ private KrollDict buildAddress(JSONObject place)
 		address.put(TiC.PROPERTY_CITY, place.optString(TiC.PROPERTY_CITY, ""));
 		address.put(TiC.PROPERTY_REGION1, ""); // AdminArea
 		address.put(TiC.PROPERTY_REGION2, ""); // SubAdminArea
-		address.put(TiC.PROPERTY_POSTAL_CODE, place.optString("zipcode", ""));
+		address.put(TiC.PROPERTY_ZIPCODE, place.optString(TiC.PROPERTY_ZIPCODE, ""));
```

Comment:
This is a breaking change. We need to retain the old values until the next major release.